crates/turbo-tasks-fs/examples/hash_glob.rs

```diff
@@ -54,18 +54,18 @@ async fn main() -> Result<()> {
 }
 
 #[turbo_tasks::function]
-pub fn empty_string() -> Vc<String> {
-    Vc::cell("".to_string())
+pub fn empty_string() -> Vc<RcStr> {
+    Vc::cell("".into())
```

Comment:
Minor cleanup opportunity: The callsite (in `hash_glob_result`) should really just use the function in `primitives.rs`